### PR TITLE
Ajax for abilities, tests

### PR DESCRIPTION
**Pull request by djuks**

### CODE REVIEW

File: features/authentication.feature [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Feature: Authentication
+
+  In order to use the website
+  As a user
+  I should be able to sign up, log in and log out
+
+Scenario: Signing up
+  Given I visit the homepage
+  When I fill in the sign up form
```
Comment (Ikera) [inline features/authentication.feature]: You are missing `Then` part in this scenario.

File: features/step_definitions/ability_steps.rb [hunk 2]
```diff
@@ -0,0 +1,69 @@
+Given("I have created several character abilities") do
+  FactoryBot.create(:character,
+                    :user => @registered_user,
+                    :name => "Trol",
+                    :description => "My first trol")
+
+end
```
Comment (Ikera) [inline features/step_definitions/ability_steps.rb]: Step title says that you create character abilities, but you create only one character.

File: features/step_definitions/ability_steps.rb [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  click_on "Trol"
+  click_on "New Ability"
+  fill_in "ability_name", :with => "Power"
+  fill_in "ability_value", :with => "10"
+  click_on "Create Ability"
+  click_on "New Ability"
+  fill_in "ability_name", :with => "Speed"
+  fill_in "ability_value", :with => "15"
+  click_on "Create Ability"
+end
```
Comment (Ikera) [inline features/step_definitions/ability_steps.rb]: In this step, you create 2 abilities, but step title says  "I visit character show page"

File: Gemfile [hunk 4]
```diff
@@ -33,12 +33,17 @@ gem 'bootsnap', '>= 1.4.2', require: false
...
   gem 'shoulda-matchers',
   git: 'https://github.com/thoughtbot/shoulda-matchers.git',
   branch: 'rails-5'
   gem 'factory_bot_rails'
+
+  gem 'rspec-core', git: 'https://github.com/rspec/rspec-core'
+  gem 'rspec-expectations', git: 'https://github.com/rspec/rspec-expectations'
+  gem 'rspec-mocks', git: 'https://github.com/rspec/rspec-mocks'
+  gem 'rspec-rails', git: 'https://github.com/rspec/rspec-rails'
+  gem 'rspec-support', git: 'https://github.com/rspec/rspec-support'
```
[the remark this answers is not in the document]
Comment (djuks) [reply Gemfile]: Yes, I have an error in tests if I put it in one gem. I found that solution online, that error is only visible in rails 6, in rails 5 works with one gem as expected.

**Pull request merged**